app/views/admin/demo_nights/index.html.erb

```diff
@@ -1,15 +1,23 @@
 <div class="active-demo-night">
-  <h1>Demo Nights</h1>
+  <h1>Active Demo Nights</h1>
+  <%= "No active demo night" if @demo_nights.currents.count == 0 %>
   <% @demo_nights.currents.each do |dn| %>
-    <%= "#{dn.name} | #{dn.status}" %>
+    <%= "#{dn.name} | #{dn.status.humanize}" %>
     <% if dn.status != "accepting_submissions" %>
-      <%= link_to "switch to accepting submissions", "#", class: "waves-effect waves-light btn turing-btn" %>
+      <%= link_to "accept submissions", admin_demo_night_path(dn, demo_night: {status: :accepting_submissions}), method: :patch, class: "waves-effect waves-light btn turing-btn" %>
```

Comment:
I don't know how I feel about this. It could be moved, but would only make the view better and the controller worse?